llvm/test/Transforms/InstCombine/stdio-custom-dl.ll

```diff
@@ -8,11 +8,10 @@ target datalayout = "e-m:o-p:40:64:64:32-i64:64-f80:128-n8:16:32:64-S128"
 @.str.1 = private unnamed_addr constant [2 x i8] c"w\00", align 1
 @.str.2 = private unnamed_addr constant [4 x i8] c"str\00", align 1
 
-; Check fwrite is generated with arguments of ptr size, not index size
```

Comment:
I'd keep this comment (as per https://llvm.org/docs/TestingGuide.html#best-practices-for-regression-tests) but maybe put it at the top of the file instead

Reply:
Well, the file could grow other stdio tests, at which point this comment would apply to only this function. I think there's a perfectly fine place for it. But if you want to do anything, use `;;` as the comment prefix to distinguish it from UTC and FileCheck directives.

Reply:
Done